.github/workflows/tox-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Semgrep Python Tests
+
+on:
+  pull_request:
+    branches: [master, develop]
+  push:
+    branches: [master, develop]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install semgrep-core from develop
+        run: sudo -E ./install-scripts/latest-artifact-for-branch.py
```

Comment:
Wont this cause issues if a PR has a breaking change in both semgrep-core and semgrep?

Reply:
yeah, but just land the core changes first 🤷 . Or ignore the build failure. Seems like a reasonable trade off vs. waiting 20 minutes for a new build to test Python changes.